avocado/core/test.py

```diff
@@ -369,6 +369,22 @@ def cache_dirs(self):
         return cache_dirs
 
     @property
+    def runner_queue(self):
+        """
+        The communication channel between test and test runner
+        """
+        return self.__runner_queue
+
+    def set_runner_queue(self, runner_queue):
+        """
+        Override the runner_queue
+        """
+        self.assertTrue(self.__runner_queue is None, "Overriding of runner_"
+                        "queue is not allowed old=%s new=%s"
```

Comment:
Here's a suggestion, `Overriding the runner queue multiple times is not allowed -> old=%s new=%s`